source/locale/pt_BR/LC_MESSAGES/internal-communication.po

```diff
@@ -720,107 +362,40 @@ msgstr ""
 
 #: ../../source/internal-communication.rst:591
 #, fuzzy
-msgid ""
-"Note: In the context of a dojot deployment the JWT Token is provided by "
-"the Auth service, and is validated by the API Gateway before redirecting "
-"the connection to the Kafka-WS. So, no validations are done by the Kafka "
-"WS."
-msgstr ""
-"Nota: No contexto de um deployment da dojot, o token é providenciado pelo"
-" serviço de Autenticação e é validado pelo Gateway API antes de "
-"redirecionar a conexão para o *Kafka-WS.* Portanto, nenhuma validação é "
-"feita pelo Kafka WS."
+msgid "Note: In the context of a dojot deployment the JWT Token is provided by the Auth service, and is validated by the API Gateway before redirecting the connection to the *Kafka-WS*. So, no validations are done by the Kafka WS."
+msgstr "Nota: No contexto de um deployment da dojot, o token é providenciado pelo serviço de Autenticação e é validado pelo Gateway API antes de redirecionar a conexão para o *Kafka-WS.* Portanto, nenhuma validação é feita pelo Kafka WS."
```

Comment:
Since the default name that you chose was Kafka WS, you should change it here too.